API/Services/Implementations/PlayerStatsService.cs

```diff
@@ -234,11 +235,11 @@ public async Task TruncateAsync()
 		return dto;
 	}
 
-	private async Task<IEnumerable<IEnumerable<MatchRatingStatsDTO>>> GetRatingStatsAsync(int playerId, int mode, DateTime dateMin, DateTime dateMax)
-	{
-		var ratingStats = await _ratingStatsRepository.GetForPlayerAsync(playerId, mode, dateMin, dateMax);
-		return _mapper.Map<IEnumerable<IEnumerable<MatchRatingStatsDTO>>>(ratingStats);
-	}
+	// private async Task<IEnumerable<IEnumerable<MatchRatingStatsDTO>>> GetRatingStatsAsync(int playerId, int mode, DateTime dateMin, DateTime dateMax)
+	// {
+	// 	var ratingStats = await _ratingStatsRepository.GetForPlayerAsync(playerId, mode, dateMin, dateMax);
+	// 	return _mapper.Map<IEnumerable<IEnumerable<MatchRatingStatsDTO>>>(ratingStats);
+	// }
```

Comment:
Is there a reason why this is being left in commented out and not just removed?

Reply:
Good eye, this was what was there before the new implementation.